src/main/scala/org/opennetworkinsight/ProxyLDA.scala

```diff
@@ -0,0 +1,31 @@
+package org.opennetworkinsight
+
+import org.opennetworkinsight.LDAArgumentParser.Config
+
+import org.apache.spark.sql.SQLContext
+import org.apache.spark.SparkContext
+import org.slf4j.Logger
+
+/**
+  * Run suspicious connections anlaysis on proxy data.
+  */
+object ProxyLDA {
+
+  def run(config: Config, sparkContext: SparkContext, sqlContext: SQLContext, logger: Logger) = {
+
+    logger.info("Proxy LDA starts")
+
+    val docWordCount = ProxyPreLDA.proxyPreLDA(config.inputPath, config.scoresFile, config.duplicationFactor, sparkContext,
+      sqlContext, logger)
+
+    val ldaResult = LDAWrapper.runLDA(docWordCount, config.modelFile, config.topicDocumentFile, config.topicWordFile,
```

Comment:
I just learned how to return a tuple instead of having to create a Map[String, RDD[String]].
That will change val ldaResult to val (documentResults, wordResults) = LDAWrapper(....)

Then in the next line (25) we can use just documentResults instead of ldaResults("document_results").

Will keep you posted.